website/content/commands/snapshot/agent.mdx

```diff
@@ -396,9 +396,16 @@ These options cannot be used when using `backup_destinations` in a config file.
 - `-azure-blob-container-name` - Container to use. Required for Azure blob storage, and setting this
   disables local storage.
 
-* `-azure-blob-environment` - Environment to use. Defaults to AZUREPUBLICCLOUD. Other valid environments
+- `-azure-blob-environment` - Environment to use. Defaults to AZUREPUBLICCLOUD. Other valid environments
   are AZURECHINACLOUD, AZUREGERMANCLOUD and AZUREUSGOVERNMENTCLOUD. Introduced in Consul 1.7.3.
 
+~> The following options were introduced in v1.20.1+ent.
+
+- `-azure-blob-service-principal-id` -
+- `-azure-blob-service-principal-secret` -
```

Comment:
These parameters are missing descriptions.

```suggestion
- `-azure-blob-service-principal-id` - The ID of the service principal that owns the blob object.
- `-azure-blob-service-principal-secret` - The secret of the service principal that owns the blob object.
```

The above text are _examples_ what should be here. Please refer to the Azure documentation to see how these parameters should be described.